src/doctor.ts

```diff
@@ -0,0 +1,66 @@
+import { getProtocolInterface } from "./deps.ts";
+
+type RuntimeVersion = {
+  name: string;
+  current: string;
+  minimum?: string;
+  error?: {
+    message: string;
+  };
+};
+
+const getHostedDenoRuntimeVersion = async (): Promise<{
+  minimum?: string;
```

Comment:
A portion of this return type can be factored out and re-used in the above `RuntimeVersion` type as well, to slightly DRY things up, in particular this:

```
{
  minimum?: string;
  error?: {
    message: string;
  }
}
```